js/obstacles.js

```diff
@@ -0,0 +1,18 @@
+class Obstacle {
+    constructor(ctx, obstaclePosx, obstaclePosy, obstacleWidth, obstacleHeight, obstacleSpeed, canvasSize) {
+        this.ctx = ctx
+        this.obstacleHeight.Size = {
+            w: obstacleWidth,
+            h: obstacleHeight
+        }
+        this.obstaclePos = {
+            x: obstaclePosx,
+            y: obstaclePosy
+        }
+        this.obstacleSpeed = obstacleSpeed
+        this.canvasSize = canvasSize
+
+        this.init()
+    }
+    init
```

Comment:
tendrias que hacer dos metodos para la clase obstaculos:

- draw(): en el que pintes un rectangulo en la posicion this.obstaclePos y de tamaño obstacleWidth y obstacleHeight
- move(): que sume la velocidad a la posicion 'y' del obstaculo. Es el mismo concepto que mover el coche